### DISCUSSION TITLE
Pagination and Teleport colliding

### DISCUSSION THREAD

**Discussion by TwilightDuck** [livewire/livewire#6935] [7. Bugs (archived)]
### Livewire version

V3.0.5

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://wirebox.app/b/xvoey

### Steps To Reproduce

Using both Teleport and pagination in the same component leads to issues. 
Using the WireBox linked above, if you comment out the "Teleport" section the code works.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment]
Thanks for reporting Nick. The problem seems to be solved when moving the teleport below the pagination. Nevertheless, this seems to be a bug. It would be of great help if you could submit a failing test. Thanks again!
